phpcs.xml.dist

```diff
@@ -14,17 +14,9 @@
 
 	<file>.</file>
 
-	<exclude-pattern>tests/*</exclude-pattern>
-	<exclude-pattern>vendor/*</exclude-pattern>
-	<exclude-pattern>vendor_prefixed/*</exclude-pattern>
-	<exclude-pattern>node_modules/*</exclude-pattern>
-	<exclude-pattern>deprecated/*</exclude-pattern>
-	<exclude-pattern>languages/*</exclude-pattern>
-	<exclude-pattern>artifact/*</exclude-pattern>
```

Comment:
I'm not sure if we want to remove `artifact` from this list. 

The `artifact` folder is created when someone creates an RC. when someone continues developing afterwards, he/she shouldn't be bothered (locally) with CS issues from that folder. 

Note that `artifact` is in the .gitignore, so on the Travis builds this will not happen.

Reply:
That explains why I couldn't find the directory. I'll add that one back in the commit which removed it.